opencti-platform/opencti-front/src/private/components/data/csvMapper/CsvMapperForm.tsx

```diff
@@ -51,29 +63,29 @@ interface CsvMapperFormProps {
 }
 
 const CsvMapperForm: FunctionComponent<CsvMapperFormProps> = ({
-  csvMapper,
-  onSubmit,
-}) => {
-  const { t } = useFormatter();
+                                                                csvMapper,
+                                                                onSubmit,
+                                                              }) => {
```

Comment:
wow what happened here ? 😅

Reply:
`66:1   error  Expected indentation of 2 spaces but found 64  @typescript-eslint/indent`
you have some lint errors in drone.

Beside that this PR seems good to me 🙌

Reply:
The format code align with the function argument position. Need to change some rules in the IDE I think.